README.md

```diff
@@ -32,10 +32,84 @@ If you make changes to the website, repeat the steps to build the image and star
 2. Run the build script: `npm run build`
 3. Upload the contents of the `./dist/` directory to the web root of your server.
 
-If you make changes to the website, repeat step 2 to build the website and upload any changed files from the `./dist/` directory to the web root of your server.
+If you make changes to the website, repeat step 2 to build the website and upload any changed files from the `./dist/`
+directory to the web root of your server.
 
-## License Information
+## Notes
 
-The Inclusive Learning Design Handbook's code is licensed under the [BSD 3-Clause](https://raw.githubusercontent.com/fluid-project/docs-inclusive-learning/blob/main/LICENSE.md) license.
+Modifications can be made to any source file or directory except for the contents of the `./dist/` directory. The
+`./dist/` directory is not versioned since it contains the built website that Eleventy generates from the source files,
+and  files in `dist` are overwritten at build time.
 
-The Inclusive Learning Design Handbook's content is licensed under the [Creative Commons Attribution 3.0](http://creativecommons.org/licenses/by/3.0/) license.
+### Trivet
+
+The ILDH website is based on [Trivet](https://github.com/fluid-project/trivet), an [Eleventy](http://11ty.dev)
+website starter kit.
+
+### Previous version of the ILDH
+
+An older version of the ILDH website used from 2011 to 2015 was previously hosted at the same repository name.
+It has been archived and can be found here:
+<https://github.com/fluid-project/2015-handbook.floeproject.org>
+
+## Working with Netlify CMS
```

Comment:
This section can be removed when you remove the admin related doc.

Reply:
I've opted to keep the section and add a note explaining how to restore this functionality. Do you think it would be better to remove it entirely?

Reply:
Let's go with your idea. Keeping this section to explain how to restore this functionality will be helpful.